src/system/Form/RadioBoxGroup.js

```diff
@@ -94,16 +95,65 @@ const RadioOption = ( {
 	);
 };
 
-RadioOption.propTypes = {
+const ChipOption = ( {
+	defaultValue,
+	option: { value, label },
+	name,
+	disabled,
+	onChangeHandler,
+} ) => {
+	const checked = `${ defaultValue }` === `${ value }`;
+
+	return (
+		<Button
+			variant="text"
+			role="radio"
+			disabled={ disabled }
+			name={ name }
+			sx={ {
+				background: checked ? 'layer.4' : undefined,
+				color: 'text',
+				minHeight: '32px',
+				px: 3,
+				boxShadow: checked ? 'low' : undefined,
+				fontWeight: 400,
+				fontSize: 2,
+				'&:hover': {
+					background: checked ? 'layer.4' : 'gray.3',
```

Comment:
We should not use `gray.3`. We should always reference a token, like layer.4. Can you check in Figma what the ideal token that references gray.3 should be? I believe it's `layer.1`.